backend/src/main/resources/db/migration/mysql/V1.3.0__setting.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE settings_dependent_settings;
+DROP TABLE setting;
+
+create table setting (
+                         is_active boolean not null,
+                         id bigint auto_increment,
+                         member_id bigint,
+                         setting_type varchar(255) check (setting_type in ('NOTIFICATION_PER_THUMBS','NOTIFICATION_PER_10_THUMBS','NOTIFICATION_PER_COMMENT','NOTIFICATION_CONSUMPTION_CONFIRMATION_REMINDING','NOTIFICATION')),
+                         primary key (id)
+);
+
+alter table setting add foreign key (member_id) references member (id);
```

Comment:
여기 뽀린키 걸때, 뽀린키 제약조건 이름을 안걸면 우리가 모르는 임의의 값이 제약조건 이름이 되서 추후에 flyway 스크립트로 뽀린키를 삭제하거나 수정할 일이 있을 때 db에서 제약조건 이름을 확인해봐야할 것 같아요. (운영 서버와 테스트 서버의 제약 조건 이름이 다를 수 있다는 것은 덤)
그래서 새로 추가되는 테이블들은 미리미리 제약 조건 이름을 정해주는게 좋겠읍니다

예시)
```suggestion
alter table setting add constraint fk_setting_member_id foreign key (member_id) references member (id);
```

<img width="735" alt="image" src="https://github.com/woowacourse-teams/2023-edonymyeon/assets/62106852/2f708521-14e1-4f48-96ac-3f7ab94de613">

Reply:
좋은 조언입니다

Reply:
@hectick 이리내씨 왜 init file 작성때 뽀링키 이름 설정 안해서 저히를 괴롭게 만드시나요 ?????